modules/oc-mirror-creating-image-set-config.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-mirroring-disconnected.adoc
+
+[id="oc-mirror-creating-image-set-config_{context}"]
+= Creating the image set configuration
+
+// TODO: Decide if this should be in a procedure step, or in its own module
+
+// TODO: Intro
+
+// Prior to running `oc mirror` / mirroring images using the `oc-mirror` plug-in, you must create the `ImageSetConfiguration` resource to define... TODO.
+
+// TODO Need more information on how this config works. Do you use the same exact config both when doing mirror2disk and disk2mirror?
```

Comment:
@afflom @jpower432 Do we expect users to use the same exact imageset-config.yaml file both when running the mirror2disk and the disk2mirror method? So they use the file to download the image set to disk, then transfer that config file to the disconnected env as well and then use it there to do disk2mirror? Or would the image set config file need to be different?

Reply:
On the disconnected side, the input would be the generated imageset. disk2mirror does not accept a config file. So instead of the flag being `--config imageset-config.yaml`, in disk2mirror operations, it would be `--from mirror_seq1_00000.tar`. It might help this think of the command as `oc mirror prefix://<destination>` with the `config` and `from` flags representing the user input artifact, be it a config or imageset. Does this help?

Reply:
Jeeze you're right, I'm not sure why I didn't notice that we weren't passing in the image set config file here! Thanks, I'll update everywhere that I made that assumption.